### ISSUE TITLE
Aria-label for Progress

### ISSUE THREAD

**Issue by laliq** [equinor/design-system#2020]

**Comment by oddvernes** [comment]
Making a properly acessible progress bar working with screen readers is a bit more elaborate. [This article](https://web.dev/building-a-loading-bar-component/) goes into how to style a native native `<progress>`  and make it accessible.

**Comment by oddvernes** [comment]
another interesting take on circular spinner using `<output>` element for announcing for screen readers: https://codepen.io/MrRoboto/pen/BaJoGpN (so no focus(), which then makes it possible for multiple spinners to exist at the same time)

And also [aria-live](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions) might be a useful in this issue

**Comment by oddvernes** [comment]
Need to do some more research on mac voiceover
